File: masterji75/RemoveDuplicate.java
class RemoveDuplicate {
    public static int removeDuplicate(int[] nums){
        int n = nums.length;
        int uniqueu_index = 0;
        for(int i =0;i<n;i++){
            if(nums[i] != nums[uniqueu_index]){
                int temp = nums[i];
                nums[i] = nums[uniqueu_index +1];
                nums[uniqueu_index + 1] = temp;
                uniqueu_index++;
            }
        }
        return uniqueu_index + 1;
    }
    public static void main(String[] args){
        int[] nums = {0,0,1,1,1,2,2,3,3,4};
        System.out.println(removeDuplicate(nums));
    }
}

File: masterji75/remove_duplicate.js
function remove_duplicate(nums) {
    var n = nums.length;
    var unique_index = 0;
    for (var i = 0; i < n; i++) {
        if (nums[i] != nums[unique_index]) {
            var temp = nums[i];
            nums[i] = nums[unique_index + 1];
            nums[unique_index + 1] = temp;
            unique_index++;
        }
    }
    return unique_index + 1;
}
var nums = [0, 0, 1, 1, 1, 2, 2, 2, 2, 3, 4, 5];
console.log(remove_duplicate(nums));
